Makefile

```
@@ -0,0 +1,10 @@
+.PHONY: ifs docker clean
```

I think that the `ifs` target is not a phony one? It references the go files

```suggestion
.PHONY: docker clean
```

---

Yes I just added it without thinking. I can remove it.